components/brave_shields/content/browser/brave_shields_util.cc

```diff
@@ -623,6 +646,37 @@ ControlType GetFingerprintingControlType(HostContentSettingsMap* map,
                                              : ControlType::BLOCK;
 }
 
+bool ShouldApplyAggressiveFingerprintingProtections(
```

Comment:
I don't think having three methods that return a bool is ideal here, but also we've been wanting to switch to useing modes from brave_shields_panel.mojom (that file should probably move/rename, but fine to use it as-is for now). I would change this to something like `brave_shields::mojom::FingerprintMode FingerprintingModeToUse(...)` or maybe `GetEffectiveFingerprintingMode`

Reply:
I think I prefer `GetEffectiveFingerprintingMode` personally